modules/org.kie.kogito.app.builder/module.yaml

```diff
@@ -20,5 +20,9 @@ artifacts:
     path: "../../../../utils"
     dest: /workspace/utils
 
+  - name: log
+    path: "../../../../log"
+    dest: /workspace/log
```

Comment:
is for write on the filesystem ?

Reply:
It's copying the new internal module to the manager's image...